go/signedexchange/cmd/gen-certurl/main.go

```diff
@@ -28,6 +31,12 @@ func run(pemFilePath, ocspFilePath, sctDirPath string) error {
 	if err != nil {
 		return err
 	}
+	if len(certs) == 0 {
+		return fmt.Errorf("Input file \"%s\" has no certificates.", pemFilePath)
```

Comment:
Use `%q` instead of `\"%s\"`

Reply:
Error messages should start with a small case: `input file ...`

Reply:
Done.